### PR TITLE
clean up danas R file

### PR DESCRIPTION
**Pull request by najisawas**
defined some copy / paste color codes as variables
clean up some documentation grammar

### CODE REVIEW

File: gwas/gwasplot_royalT.R [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
+library(dplyr)
+
 # Using ggplot2 for GWAS by Dana Al-Hindi. 
-# Name Royal T because of the Royal Tenenbaum colors scheme.
+# The file is named Royal T after the Royal Tenenbaum colors scheme.
+
 # This code is set up to plot the statistical results from a GWAS. 
 # It requires a data frame that has a column CHR and BP present. 
-# arg[1] is the name of the file, arg[2] is the column of which to plot (p-values)
+# args[1]: the name of the file
```
Comment (dralhindi) [inline gwas/gwasplot_royalT.R:9]: this is shit.

Comment (dralhindi) [inline gwas/gwasplot_royalT.R:9]: sorry, this is utter shit.

File: gwas/gwasplot_royalT.R [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
-            "#9A872D", "#F5CDB6", "#F7B0AA", "#FDDDA4", "#76A08A", 
-            "#9A872D", "#F5CDB6", "#F7B0AA", "#FDDDA4", "#76A08A",
-            "#9A872D", "#F5CDB6", "#F7B0AA")
+# Set colors for plot.
+
+olive_yellow <- "#9A872D"
+soft_orange <- "#F5CDB6"
+soft_red <- "#F7B0AA"
+mango_yogurt <- "#FDDDA4"
+mostly_desaturated_dark_cyan_lime_green <- "#76A08A"
```
Comment (dralhindi) [inline gwas/gwasplot_royalT.R]: please simplify color name

**Review by dralhindi — COMMENTED** [review]
this is shit. great. hard. smelly. shit.

File: gwas/gwasplot_royalT.R (hunk 1 above)
Comment (najisawas) [reply gwas/gwasplot_royalT.R:9]: wow i tried so hard 😭

File: gwas/gwasplot_royalT.R (hunk 2 above)
Comment (najisawas) [reply gwas/gwasplot_royalT.R]: changed, thanks!

**Review by dralhindi — APPROVED** [review]

**Pull request merged**